beeline/src/java/org/apache/hive/beeline/JSONFileOutputFormat.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * This source file is based on code taken from SQLLine 1.9
+ * See SQLLine notice in LICENSE
+ */
+package org.apache.hive.beeline;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.util.MinimalPrettyPrinter;
+
+/**
+ * OutputFormat for hive JSON file format
+ * Removes "{ "resultset": [...] }" wrapping and prints one object per line
+ * 
```

Comment:
Nit: The Hive style guidelines require a period at the end of the first sentence here.

Please provide an example and provide some additional context... something like...

```
This output format matches the same format as a Hive table created with JSONFILE file format:

CREATE TABLE ... STORED AS JSONFILE;
```

Then provide a sample output in the JavaDoc.  Feel free to use HTML tags and JavaDoc docs to format and enhance.